circle.yml

```diff
@@ -1,4 +1,6 @@
 machine:
+  python:
+    version: 3.5.1
```

Comment:
Nope. CI build goes from 1'30 to 4'30 just to download and set up Python & friends.